File: in_Python/0709 To Lower Case.py
class Solution(object):
    def toLowerCase(self, str):
        """
        :type str: str
        :rtype: str
        """
        #return str.lower()
        res = ''
        up = ord('Z')
        low = ord('A')
        m = ord('z')
        for c in str:
            x = ord(c)
            if low <= x <= up:
                res += chr(x - up + m)
            else:
                res += c
        return res

File: in_Java/0709 To Lower Case.java
class Solution {
    public String toLowerCase(String str) {
        char[] ch = str.toCharArray();
        for(int i = 0; i < ch.length; i++){
            char c = ch[i];
            if(c >= 'A' && c <= 'Z'){
                ch[i] = Character.toLowerCase(c);
            }
        }
        return String.valueOf(ch);
    }
}